src/style.css

```
@@ -0,0 +1,102 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+  border: none;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: 300;
+  box-sizing: border-box;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  outline: none;
+  font-weight: 800;
+}
+
+.input::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
+  font-family: inherit;
+  font-size: 16px;
+}
+
+.img {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.input--small {
+  height: 42px;
+  padding: 10px 33px;
+  font-family: inherit;
+  font-size: 14px;
+  font-weight: 400;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  /* 400 */
+  background: #fff;
+}
+
+.input--small:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  /* font-weight: bold; */
+}
+
+.input--small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
```

do u really need to add the same value with hover and without hover?

```suggestion
  border-radius: 4px;
```